src/constructs/ec2/security-groups.ts

```diff
@@ -70,3 +70,18 @@ export class GuPublicInternetAccessSecurityGroup extends GuSecurityGroup {
     });
   }
 }
+
+export class GuHTTPSEgressSecurityGroup extends GuSecurityGroup {
```

Comment:
Not sure if there is a proper convention for cases like this, but I find `GuHttpsEgressSecurityGroup` slightly more readable. Feel free to ignore without comment if you disagree!

Reply:
Done!